dotCMS/src/integration-test/java/com/dotmarketing/db/DockerSecretDataSourceStrategyTest.java

```diff
@@ -0,0 +1,125 @@
+package com.dotmarketing.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.repackage.org.apache.commons.io.FileUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotcms.util.TestInitialContext;
+import com.dotmarketing.util.Constants;
+import com.liferay.util.SystemEnvironmentProperties;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import java.io.File;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/**
+ * Tests for {@link DockerSecretDataSourceStrategy}
+ * @author nollymar
+ */
+public class DockerSecretDataSourceStrategyTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void testApply() throws SQLException, IOException, NamingException {
```

Comment:
what is the expected result of this test?

Reply:
Done